r/tests/testthat/test-compute-aggregate.R

```diff
@@ -267,6 +260,20 @@ test_that("quantile and median NAs, edge cases, and exceptions", {
   )
 })
 
+test_that("median passes ... args to quantile", {
+  skip_if(
+    !"..." %in% names(formals(median)),
+    "The median generic lacks dots in R 3.3.0 and earlier"
+  )
```

Comment:
I'm surprised that `R CMD check` did not detect S3 generic/method consistency problems in R 3.3.0, but it didn't:
https://github.com/ursacomputing/crossbow/runs/2275559899#step:7:193
Given that, it's simpler to skip the tests that use the `median()` dots args than it would be to conditionally define `median.ArrowDatum` with/without dots depending on whether the generic has them.